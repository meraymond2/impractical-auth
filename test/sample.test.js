import React from 'react';
import InputCapture from '../dev/inputCapture.jsx';
import renderer from 'react-test-renderer';

test('Do the thing.', () => {
  const component = renderer.create(
    <InputCapture />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  console.log("");
  console.log(tree);
  console.log("");
  // // manually trigger the callback
  // tree.props.onMouseEnter();
  // // re-rendering
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();

  // // manually trigger the callback
  // tree.props.onMouseLeave();
  // // re-rendering
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
});

test('adds 1 + 2 to equal 3', () => {
  expect(1 + 2).toBe(3);
});