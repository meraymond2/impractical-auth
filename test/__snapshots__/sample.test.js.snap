// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Do the thing. 1`] = `
<p
  onClick={[Function]}
>
  I'm going to be the input capture thingy...
</p>
`;
